### ISSUE TITLE
Upgrade openvpn to 2.5.x to manage data-ciphers options

### ISSUE THREAD

**Issue by the-lazy-fox** [dperson/openvpn-client#392]
Hi,

Some VPN now require OpenVPN to be in version 2.5.x so data-ciphers options are managed.
Could you please upgrade it in the container?

Thanks

**Comment by grbharati** [comment]
yes getting error because of the ciphers. Can we update the openvpn?
![image thumb png 3697bf003a2ba98c95863c935d356fbe](https://user-images.githubusercontent.com/69709740/147377285-be1012fa-c754-40bd-83ad-6d36f06724a6.png)

**Comment by avpnusr** [comment]
I don't want to advertise another docker-image, but I forked this a while ago and set up an own build-chain with weekly updates. Standard "latest"-image is openvpn 2.5, there is a specific tag, if you need to continue using openvpn 2.4. All the rest is unchanged, so all kudos to @dperson ;) If it is unwanted, to reference this as a workaround, until the update, feel free to delete my answer here.

Alternative-Image: https://hub.docker.com/r/avpnusr/docker-openvpn

Regards,

avpnusr

**Comment by dudul** [comment]
@avpnusr do you have a git repository for this fork?

**Comment by avpnusr** [comment]
Sure, had it on my own gitlab, but made the changes to the fork here on github ;)
https://github.com/avpnusr/openvpn-client

**Comment by SylvainAssemat** [comment]
Thanks for this fork (2.5)
I am stuck on compute engine when i try to use it

```
Linux compute-i2 3.10.0-1160.49.1.el7.x86_64 #1 SMP Tue Nov 30 15:51:32 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
LSB Version:    :core-4.1-amd64:core-4.1-noarch
Distributor ID: CentOS
Description:    CentOS Linux release 7.9.2009 (Core)
Release:        7.9.2009
Codename:       Core
```


`manifest for docker.io/avpnusr/docker-openvpn:latest not found`

Could you help me on this specific case please ? 
Maybe a multiplatform build problem ?

Thanks 

(on my PC everything work fine)

**Comment by scruel** [comment]
@avpnusr With your fork, still can't run:
Options error: --lport and --nobind don't make sense when used together
![image](https://github.com/dperson/openvpn-client/assets/16933298/2a0defb1-615b-4531-aadb-3d95f353effa)

**Comment by scruel** [comment]
I think we will need to update openvpn to 3.x now.
